cmd/destroy/destroy.go

```diff
@@ -167,12 +169,14 @@ func Destroy(ctx context.Context) *cobra.Command {
 				k8sClientProvider: okteto.NewK8sClientProvider(),
 			}
 
-			kubeconfigPath := deploy.GetTempKubeConfigFile(name)
+			kubeconfigPath := getTempKubeConfigFile(name)
```

Comment:
@ifbyol Why are we creating the `tempkubeconfig` for destroy? It's not using the proxy. Could we remove it and use directly the context kubeconfig?

Reply:
We are creating the temporal kubeconfig to avoid to change the user's kubeconfig. Bear in mind that we are setting the okteto context in the kubeconfig. But the user's current k8s context can be a different one. 

I would not remove the temporal file to avoid the side effect of changing user's kubeconfig or use the wrong kubeconfig